### DISCUSSION TITLE
KB5030310 caused black screen after log in

### DISCUSSION THREAD

**Discussion by lkasahi** [valinet/ExplorerPatcher#2215] [General]
Hi all. Sorry to bother for the black screen issue after updating Windows again. 

I just installed KB5030310 released a few days ago and black screen occured yet again. This time, task manager can be opened but both _cmd_ and _ms-config:_ were unable to run inside task manager as I tried to uninstall EP. I then got into safe boot mode by running _msconfig_, flicked minimal safe boot option on, and rebooting afterwards. After a successful reboot, EP prompted a sweet window, saying that explorer.exe has crashed 3 times in 10 seconds upon boot up, and provided an option to uninstall directly, which solved the issue.

I didn't note the version of the app and symbol I was using, but I always update them as soon as I received a notification so I assume they were both the latest version. I also opted out modifying the right click menu of the task bar months ago, so this might be a new issue.

Event manager recored errors regarding the issue, don't know if this helps:
AppName: Explorer.EXE 
AppVersion: 10.0.22621.2361 
AppTimeStamp: 480de713 
ModuleName: dxgi.dll 
ModuleVersion: 22621.1992.56.3 
ModuleTimeStamp: 64e6228c 
ExceptionCode: c0000005 
FaultingOffset: 000000000003ad19 
ProcessId: 0x24ac 
ProcessCreationTime: 0x1d9f2c0e9068341 
AppPath: C:\WINDOWS\Explorer.EXE 
ModulePath: C:\WINDOWS\dxgi.dll 
IntegratorReportId: c89b2200-a94f-4fbd-bf7a-7ee44ac00381 
PackageFullName:
PackageRelativeAppId:

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-us/topic/september-26-2023-kb5030310-os-build-22621-2361-preview-363ac1ae-6ea8-41b3-b3cc-22a2a5682faf

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Comment by pyrates999** [comment]
You also installed an old version of EP.  After you've uninstalled the preview update, download the latest version of EP and install that.
